### DISCUSSION TITLE
converting abaqus mesh (.inp) to .e

### DISCUSSION THREAD

**Discussion by Traiwit** [idaholab/moose#18147] [Q&A Tools] [answered]
Hi guys,

Has anyone ever tried to convert .inp to .e via MOOSE?

I tried to follow the example in test/tests/mesh/abaqus_input

However, the mesh (.e) I obtained doesn't look correct as some elements are missing (attached - paraview)

Not too sure what is the problem, .inp looks fine when I view it on Cubit/Abaqus

Please let me know if you have any idea.

Note: I also tried to convert .inp to .e using meshio, also doesn't work as well.

Kind regards,
Traiwit

![2021-06-23_15-31](https://user-images.githubusercontent.com/79562439/123041424-9f409900-d438-11eb-92b3-49e1e3766010.png)

**Comment by GiudGiud** [comment]
I have not. Can you convert it via Paraview instead of via MOOSE?
There's got to be some format that is common between Paraview and Abaqus, maybe not inp. Maybe .nas from some quick googling

**Reply by Traiwit** [reply]
unfortunately Paraview doesnt read .inp file

**Reply by GiudGiud** [reply]
Is there another format Abaqus can export that paraview can read?

**Comment by Traiwit** [comment, marked as answer]
@GiudGiud I figured out now, so MOOSE doesn't read some of Abaqus format

for example

original .inp file has

```
*ELSET, ELSET=DRV_Y2026_M03, GENERATE
 541205, 541208
 666835, 666839
 691109, 691112
```

it should read from element 541205 to 541208, but instead MOOSE only read 2 elements (541205 and 541208).

I have to reformat .inp to 

```
*ELSET, ELSET=DRV_Y2026_M03, GENERATE
 541205, 541208,1
 666835, 666839,1
 691109, 691112,1
```

so now MOOSE read from element 541205 to 541208

![2021-06-25_15-10](https://user-images.githubusercontent.com/79562439/123372779-8ca89a00-d5c7-11eb-9984-327d1286d6cf.png)
